cmd/siot/main.go

```diff
@@ -22,7 +22,7 @@ func main() {
 	flag.Parse()
 
 	if *flagSim {
-		sim.DeviceSim(*flagSimPortal, *flagSimDeviceID)
+		go sim.DeviceSim(*flagSimPortal, *flagSimDeviceID)
```

Comment:
I like that for development -- eventually we may want to re-look at the initial check out and test experience. But perhaps, the demo experience should be to start the server, and then send individual samples to the server using the SIOT cmd line. This would actually be something that is useful.